_posts/tr/pages/2017-01-01-download.md

```diff
@@ -0,0 +1,169 @@
+---
+name: download
+title: İndir
+permalink: /tr/download/
+type: pages
+layout: page
+lang: tr
+version: 1
+
+## These strings need to be localized.  In the listing below, the
+## comment above each entry contains the English text.  The key before the
+## colon must not be changed; the value after the colon should be the
+## translation.  For example (Spanish):
+##
+##     ## title: Download - Bitcoin
+##     title: Descargar - Bitcoin
+# title: Download - Bitcoin
+title: Bitcoin - İndir
+# latestversion: "Latest version:"
+latestversion: "Güncel Version:"
+# download: "Download Bitcoin Core"
+download: "Bitcoin Core' u İndir"
+# downloados: "Or choose your operating system"
+downloados: "İşletim sistemini seç"
+# download_sha: "SHA256 binary hashes"
+download_sha: "SHA256 binary hashes"
+# download_sig: "SHA256 hash signatures"
+download_sig: "SHA256 hash signatures"
+# downloadtorrent: "Download torrent"
+downloadtorrent: "Torrent indir"
+# source: "Source code"
+source: "Kaynak kodu"
+# versionhistory: "Show version history"
+versionhistory: "Version güncellemelerini göster"
+# notelicense: "Bitcoin Core is a community-driven <a href=\"https://www.fsf.org/about/what-is-free-software\">free software</a> project, released under the open source <a href=\"http://opensource.org/licenses/mit-license.php\">MIT license</a>."
+notelicense: "Bitcoin Core, açık kaynaklı <a href=\"http://opensource.org/licenses/mit-license.php\">MIT lisansı</a> altında yayınlanan, topluluk odaklı bir <a href=\"https://www.fsf.org/about/what-is-free-software\">ücretsiz yazılım</a> projesidir."
+
+
+# notesync: >
+#   Bitcoin Core requires a one-time download of about $(DATADIR_SIZE)GB
+#   of data plus a further $(MONTHLY_RANGE_GB)GB per month.  By default,
+#   you will need to store all of that data, but if you <a
+#   href="https://bitcoin.org/en/full-node#reduce-storage">enable
+#   pruning</a>, you can store as little as $(PRUNED_SIZE)GB total without
+#   sacrificing any security.
+notesync: >
+   Bitcoin Core, bir kerelik yaklaşık $(DATADIR_SIZE) GB veri ve ayrıca ayda $(MONTHLY_RANGE_GB) GB daha indirmeyi gerektirir. 
+   Varsayılan olarak, tüm bu verileri depolamanız gerekecek, ancak <a
+   href="https://bitcoin.org/en/full-node#reduce-storage">GB kısıtlanmasını etkinleştirirseniz</a>, 
+   herhangi bir güvenlikten ödün vermeden toplamda $(PRUNED_SIZE) GB' a kadar az depolayabilirsiniz.
+# full_node_guide: "For more information about setting up Bitcoin Core, please read the <a href=\"https://bitcoin.org/en/full-node\">full node guide</a>."
+full_node_guide: "Bitcoin Core kurulumu hakkında daha fazla bilgi için lütfen <a href=\"https://bitcoin.org/en/full-node\">tam çözüm kılavuzunu</a> okuyun."
+
+
+
+# patient: "Check your bandwidth and space"
+patient: "Bant genişliğinizi ve alanınızı kontrol edin"
+
+pgp_key_fingerprint: "PGP anahtar parmak izi"
+verify_download: "İndirmenizi doğrulayın"
+
+verification_recommended: >
```

Comment:
I wonder why can't I see the untranslated paragraphs that start here. 🤔

Reply:
Download page completely translated in Turkish language (131d121)

Reply:
I understand now, that coulda happened caused because of the
wrongly used version number fixed in 131d1219447574e 👌👌👌

Reply:
Absolutely 🤗

Reply:
indeed, just by setting the version 5 on top of that rev half translated,
the english untranslated paragraphs popped up 👌👌👌